starlite/openapi/parameters.py

```diff
@@ -115,6 +116,64 @@ def create_parameter(
     )
 
 
+def _get_unlayered_params(
```

Comment:
`get_recursive_handler_parameter()`?

We should use the full word `parameter` in these names to match the other function names in the module.

Also, don't worry about making these private functions as we wouldn't consider the rest of this module public interface either so no need to discriminate these.

Reply:
>Also, don't worry about making these private functions as we wouldn't consider the rest of this module public interface either so no need to discriminate these.

it's more of a personal thing that i do without thinking since i usually name all utility functions that way in order to visually separate them. i'll make them public